src/api/test/functional/source_services_test.rb

```diff
@@ -260,6 +247,64 @@ def test_run_source_service
     assert_response 404
   end
 
+  def test_service_merge
+    login_tom
+    raw_put '/source/home:tom/service/_meta', "<package project='home:tom' name='service'> <title /> <description /> </package>"
```

Comment:
Can you write a comment that this is just the setup of the test (I guess it is...)